jfr-daemon/src/main/java/com/newrelic/jfr/daemon/DaemonConfig.java

```diff
@@ -17,12 +17,18 @@
 
   static final int DEFAULT_JMX_PORT = 1099;
 
+  private static final String DEFAULT_DAEMON_VERSION = "UNKNOWN-VERSION";
   private static final String DEFAULT_JMX_HOST = "localhost";
   private static final boolean DEFAULT_USE_SHARED_FILESYSTEM = false;
   private static final boolean DEFAULT_AUDIT_LOGGING = false;
   private static final boolean DEFAULT_USE_LICENSE_KEY = false;
   private static final Duration DEFAULT_HARVEST_INTERVAL = Duration.ofSeconds(10);
   private static final String DEFAULT_MONITORED_APP_NAME = "My Application";
+  private static final String DEFAULT_PROXY_HOST = null;
+  private static final Integer DEFAULT_PROXY_PORT = null;
+  private static final String DEFAULT_PROXY_SCHEME = null;
```

Comment:
Nit: final fields should not be set to null.

Reply:
I made these final just because they're meant to be default values which don't change.

Reply:
I made these final just because they're meant to be default values which don't change.